app/tests/integration/nimbus/conftest.py

```diff
@@ -246,6 +246,7 @@ def _create_experiment(selenium):
         overview.select_risk_brand_false()
         overview.select_risk_revenue_false()
         overview.select_risk_partner_false()
+        overview.projects = ["Advanced hybrid forecast"]
```

Comment:
You can't hardcore a name like this, they'll be different every time.

Reply:
Ugh, is there a way to add a test project that will always be there?

Reply:
In theory, this is why I was asking about how to trigger the multiselect.  Why can't we just hit it and select the first option?

Reply:
Could also do it the same way we do for countries/locales/languages, except instead of reading the fixture file off the db, query it from the gql to get the ids.  Actually we could/should be doing that for countries/locales/languages as well.  Reading off those fixture files is messy.

Reply:
Actually yeah that's just easier, let's do that.

https://github.com/mozilla/experimenter/issues/8196